pyproject.toml

```diff
@@ -39,19 +40,45 @@ extra_standard_library = [
     "UserString",
 ]
 
+[tool.pyright]
```

Comment:
The tests show several of these `reportUnsupportedDunderAll` diagnostics from pyright. Are they bugs in the stubs? Should we suppress the warning or fix them?

```text
"relativedelta" is specified in __all__ but is not present in module (reportUnsupportedDunderAll)
```

Reply:
Done in https://github.com/microsoft/python-type-stubs/pull/335